app/src/main/java/com/example/taskyy/ui/enums/EditTextScreenType.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.taskyy.ui.enums
+
+enum class EditTextScreenType {
+    EDIT_DETAILS,
+    EDIT_TITLE,
+    NULL
```

Comment:
This null value doesn't really make sense. Kotlin lets you model anything as a null value, so you don't need your own ones. The enum is fine with just the first two options. To indicate there is no value, you can still do 
```
val myEnum: EditTextScreenType? = null
```